api/python/cellxgene_census/src/cellxgene_census/experimental/ml/pytorch.py

```diff
@@ -657,10 +656,9 @@ def shape(self) -> Tuple[int, int]:
 
     @property
     def obs_encoders(self) -> Encoders:
-        """
-        Returns a dictionary of :class:`sklearn.preprocessing.LabelEncoder` objects, keyed on ``obs`` column names,
+        """Returns a dictionary of :class:`sklearn.preprocessing.LabelEncoder` objects, keyed on ``obs`` column names,
         which were used to encode the ``obs`` column values. These encoders can be used to decode the encoded values as
-        follows:
+        follows.
```

Comment:
Are those changes introduced by ruff? I feel like the colon is clearer.

Reply:
It was a manual fix since Ruff was complaining about the line not ending in a period. I can revert this

Reply:
Resolved by adding a new line